packages/sms/sms-manager.ts

```diff
@@ -37,7 +37,8 @@ const handleSendingSMS = ({
       if (!team?.parent?.isOrganization) return;
 
       await checkSMSRateLimit({ identifier: `handleSendingSMS:team:${teamId}`, rateLimitingType: "sms" });
-      const sms = twilio.sendSMS(reminderPhone, smsMessage, senderID, teamId);
+      const sanitizedMessage = smsMessage.replace(/&#x2F;/g, "/");
```

Comment:
Good find.  Let's add a ticket to refactor this afterwards. I would like to find the place where the mis-encoding is happening.